src/lib/libhtml5_webgl.js

```diff
@@ -109,7 +109,7 @@ var LibraryHtml5WebGL = {
 #endif
 
     var canvas = findCanvasEventTarget(target);
-    if (canvas.canvas) { canvas = canvas.canvas; }
+    if (canvas && canvas.canvas) { canvas = canvas.canvas; }
```

Comment:
I think you can write `canvas?.canvas` instead of `canvas && canvas.canvas`?

Reply:
Yes, that's true. I'll fix it.